docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[endgame-sensor-full-disk-access]]
+= Enable Full Disk Access for the Endgame sensor
+
+Elastic Endgame requires Full Disk Access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. This means you need to manually grant permission for the Endgame sensor to access these protected areas of your Mac.
+
+[discrete]
+[[macos-permissions]]
+== macOS permissions
+
+The behavior of the Endgame sensor and Elastic Endpoint differs based on your macOS version. MDM/JAMF users can pre-approve all Full Disk Access without granting permission to the sensor. However, depending on the macOS version and sensor type, non-MDM/JAMF users may be prompted to enable Full Disk Access for required security files.
+
+Here are the following Full Disk Access requirements for macOS:
+
+- `10.13, 10.14, 10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<kernel-ext-approval-endgame, kernel extension>>. During installation, you'll be prompted to go to **System Preferences** and approve loading the kernel. Upon approval, installation proceeds.
+
+- `11.0`, `12.0`: Users cannot proceed with installation without first granting the sensor the ability to load a <<system-extension, system extension>>. During installation, you will be prompted to go to **System Preferences** and approve loading the system extension. Upon approval, a second prompt appears to enable **Network Filtering**. Approve this final prompt for installation to proceed.
++
+You also must grant Full Disk Access to `com.endgame.systemextension`.
+
+- `10.14.6+, 10.15, 11.0, 12.0`: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Endgame sensor only. To see Full Disk Access requirements for the {elastic-endpoint}, refer to <<enable-fda-endpoint>>.
+
+[discrete]
+[[kernel-ext-approval-endgame]]
+== Approve the kernel extension for the Endgame sensor
+
+For endpoints running macOS Mojave (10.14) and earlier, {elastic-endpoint} will attempt to load a kernel extension (as opposed to a system extension) during installation. This kernel extension is required to provide insight into system events such as process events, file system events, and network events. The following prompt appears during installation:
```

Comment:
We need to be very explicit that this ONLY applies to 7.15 and below. Since this is docs for 8.0, should we just drop this section entirely @ferullo?